### ISSUE TITLE
looks like the html_output_folder_path_str is no longer being obeyed for some static files

### ISSUE THREAD

**Issue by jcolson** [obsidian-html/obsidian-html#385]
in my config.yml:

```
html_output_folder_path_str: 'output/html'
```
search-json.gzip showing `/output/obs.html/...` - should be `/output/html/obs.html/...`
output for static stuff from firefox:
<img width="604" alt="image" src="https://user-images.githubusercontent.com/2362647/189153785-f8b9072a-ed61-41ee-85b1-b6f92cd82e7d.png">

and 3-force-graph.js showing `/obs.html/...` - should be `/output/html/obs.html/...`

<img width="526" alt="image" src="https://user-images.githubusercontent.com/2362647/189154694-09d59e1b-6e8d-49c4-b61f-ff71859e48c1.png">

webserver showing the 404's
```
[2022-09-08 15:46:36] ERROR `/output/obs.html/data/search.json.gzip' not found.
[2022-09-08 15:48:00] ERROR `/obs.html/static/3d-force-graph.js' not found.
[2022-09-08 15:55:28] ERROR `/output/html/TTRPG/obs.html/data/search.json.gzip' not found.
[2022-09-08 15:55:29] ERROR `/output/html/TTRPG/obs.html/data/search.json.gzip' not found.
[2022-09-08 15:55:31] ERROR `/output/obs.html/data/search.json.gzip' not found.
[2022-09-08 15:55:45] ERROR `/output/obs.html/data/search.json.gzip' not found.
[2022-09-08 15:56:08] ERROR `/obs.html/static/3d-force-graph.js' not found.
[2022-09-08 15:56:08] ERROR `/output/html/obs.html/obs.html/data/search.json.gzip' not found.
```

**Comment by dwrolvink** [comment]
this should be fixed by #386 
I'll test more tomorrow, but I am hopeful.

**Comment by jcolson** [comment]
this looks to be addressed in the packaged release tag v3.2.0 - thanks!
